src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/Profile.java

```diff
@@ -180,76 +157,12 @@ public void checkWeightInRange(Criterion crit, LinearAVF linearAVF) {
    * @param crit the criterion we want to know the value
    * @return the subjective value weight
```

Comment:
Il n'y a pas de retour ici, puisque la fonction renvoie un void

Reply:
Fix in a4f838f54a00fc6495e56771eb0771b8e6c76c2f